### PR TITLE
WIP: replacer deployment service

### PR DESCRIPTION
**Pull request by rvantonder**

### CODE REVIEW

File: base/replacer/replacer.Deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          failureThreshold: 1
+          httpGet:
+            path: /healthz
+            port: http
+            scheme: HTTP
+          periodSeconds: 1
+        resources:
+          limits:
+            cpu: "4"
+            memory: 500M
```
Comment (rvantonder) [inline base/replacer/replacer.Deployment.yaml]: Replacer is CPU heavy so I'm choosing 4 for cpu (cf. 2 for searcher). Reasonable? Needs very little memory, so choosing 500M (cf. 1G for searcher).

File: base/replacer/replacer.Deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - env:
+        - name: REPLACER_CACHE_SIZE_MB
+          value: "100000"
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: CACHE_DIR
+          value: /mnt/cache/$(POD_NAME)
+        image: index.docker.io/sourcegraph/replacer:insiders@sha256:7c376f1983652dbb899ccb72709c01bb68bcefdad488cf3bae153b2d3f3bb9f7
```
Comment (rvantonder) [inline base/replacer/replacer.Deployment.yaml]: Replacer has been pushed with an `insiders` tag. I got the hash above by running: `docker inspect --format='{{.RepoDigests}}' sourcegraph/replacer:insiders`.

Is this fine?

Comment (ggilmore) [reply base/replacer/replacer.Deployment.yaml]: Yes, that's exactly what Renovate is doing.

File: base/replacer/replacer.Service.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "6060"
+    prometheus.io/scrape: "true"
+  labels:
+    app: searcher
```
Comment (ggilmore) [inline base/replacer/replacer.Service.yaml]:
```suggestion
    app: replacer
```

File: base/replacer/replacer.Service.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    deploy: sourcegraph
+  name: searcher
```
Comment (ggilmore) [inline base/replacer/replacer.Service.yaml]:
```suggestion
  name: replacer
```

File: base/replacer/replacer.Deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        terminationMessagePolicy: FallbackToLogsOnError
+        name: searcher
```
Comment (ggilmore) [inline base/replacer/replacer.Deployment.yaml]:
```suggestion
        name: replacer
```

**Pull request merged**